examples/studio.html

```diff
@@ -159,6 +159,9 @@
 const localLine = new THREE.Line3();
 const localRaycaster = new THREE.Raycaster();
 
+window.addEventListener('mouseup', e => {
```

Comment:
This might need to be moved to a more appropriate location

Reply:
I think the handler is fine, but the way the transform controls work is that we create/destroy the object when the controls appear.

I think we should unify under a single path rather than add a new mode for `transformEnabled`. I can't convince myself this is free of corner cases otherwise.

Does calling `_removeTransformControls` work instead?

Reply:
What happens in that case is the `transformControls` are removed, but the iframe still moves when the mouse cursor is moved back in the viewport.

Reply:
Hm, if it's the transform controls that are doing that, and destroying them does not disable them, then I think that's a bug in the transform controls destruction/timing.

I would look into why the events continue to be handled for a dead transform controls in that case.

Reply:
Code pointer: https://github.com/exokitxr/exokit/pull/1065/files#diff-5f6b5375e965927cdac457568760b24aR1124

Reply:
got it:

- studio.html disposes of transformControls - https://github.com/exokitxr/exokit/blob/studio/examples/studio.html#L1118
-  disposing of transformControls removes mouseup listener - https://github.com/exokitxr/exokit/blob/studio/examples/TransformControls.js#L126
- mouseup listener is the only way that would document.removeEventListener( "mousemove", onPointerMove, false );

so simply adding to the dispose in TransformControls.js:
document.removeEventListener( "mousemove", onPointerMove, false );

fixes it